app/src/main/scala/org/alephium/explorer/persistence/queries/BlockQueries.scala

```diff
@@ -52,7 +52,7 @@ object BlockQueries extends StrictLogging {
         queryName  = "mainChainQuery",
         queryInput = "Unit",
         explain    = explain,
-        message    = None,
+        message    = Iterable.empty,
```

Comment:
oh I see you change here, but still can we have multiple messages? if yes we could renamed the field to `messages`, otherwise we should use `Option`